### ISSUE TITLE
Slack

### ISSUE THREAD

**Issue by stephanstapel** [stephanstapel/ZUGFeRD-csharp#352]
From the announcement email:

Profil EXTENDED: Einführung von zulässigen Rundungsungenauigkeiten (englisch „Slack“). Somit führen geringe Abweichungen bei den ausgewiesenen Steuerbeträgen nicht mehr automatisch zu einem Fehler. Als gutes Beispiel kann ein Bruttopreis von 9,99 Euro herangezogen werden, der sich nicht als Nettopreis mit zwei Nachkommastellen und 19 % Umsatzsteuer abbilden lässt. Ein solcher Slack wird auch im Rahmen der Überarbeitung der EU-Norm zur elektronischen Rechnung thematisiert, kann jedoch vorerst nur im Profil EXTENDED genutzt werden.

**Comment by CC-DAR** [comment]
Hi, gibt es dazu Updates? Da wir in unserer Software von Bruttopreisen ausgehen, haben wir öfters Probleme mit Rundungsdifferenzen....

**Comment by stephanstapel** [comment]
Hallo @CC-DAR ,
ich würde das Feature im Moment von mir aus nicht implementieren. Ich freue mich darauf, einen Pull Request zu reviewen. Alternativ auch gerne per Consulting, siehe Startseite.

**Comment by Tragen** [comment]
Welches Feld ist das?
Hast du einen Hinweis auf die Doku?

**Comment by stephanstapel** [comment]
Such mal im Technischen Anhang des Extended-Profiles nach RoundingAmount

**Comment by stephanstapel** [comment]
![Image](https://github.com/user-attachments/assets/b7407a75-d8ff-4c57-b469-06ff4623b820)

**Comment by stephanstapel** [comment]
Mein Ziel ist nur, den Standard abzubilden. Ich hatte alle neuen Features, die noch nicht implementiert sind, als Issue angelegt.

**Comment by Tragen** [comment]
Wenn ich in die Klasse instanziere kann ich .RoundingAmount setzen. Getestet hab ich es noch nicht.
Was fehlt denn noch?

**Comment by stephanstapel** [comment]
Ja, Du hast recht. Wahrscheinlich kann ich das Ticket schließen :)
Hast Du es schonmal eingesetzt und kannst mir Beträge nennen, mit denen es erfolgreich validiert? Dann würde ich noch ein Stück Text im Readme schreiben für diejenigen, die unter der limitierten Stellenanzahl leiden.

**Comment by Tragen** [comment]
Wir sind gerade dabei das einzubauen. Ich kann dir da den Erfolg nächste Woche posten.

**Comment by Tragen** [comment]
Also nach einigen Tests ist das ganze einfach nicht umsetzbar.
Wenn der Basispreis 4 Nachkommastellen hat mit Menge und Rabatt kann ich den Positionspreis nicht mit 2 Nachkommastellen eintragen da der Rabatt auch nur 2 Nachkommastellen hat. Damit komme ich nie mit den Preisen hin und die Validatoren bringen alle nur Fehlermeldungen.
Die Lösung war die Aufschläge und Rabatte auf 0 zu setzen und einfach mit 2 Nachkommastellen alles anzugeben und im Info-Feld geben wir mit welche Preise es ursprünglich waren.
